projects/laji/src/app/shared-modules/observation-result/selected-field-group/selected-field-group.component.ts

```diff
@@ -41,48 +41,33 @@ export class SelectedFieldGroupComponent {
     }
   }
 
-  onMoveUp(field: string | string[]) {
-    if (Array.isArray(field)) {
-      if (this.selected.indexOf(field[0]) > 0) {
-        field.forEach(column => {
-          this.moveUp.emit(column);
-
-          if (this.columnSelector) {
-            this.columnSelector.moveFieldByName(column, -1);
-          }
-        });
-      }
-      return;
-    }
-
-    this.moveUp.emit(field);
+  onMoveUp(field: string[]) {
+    const indexOfField = this.selected.indexOf(field[0]);
+    const widthOfNextGroup = this.getFieldColumnArrayLength(this.selected[indexOfField - 1]);
```

Comment:
Minor nit: move one row below since not needed yet (less computing is good always. Also, in general in similar cases as this, computing variables only when after all sanity checks are done results in less errors, e.g. `this.getFieldColumnArrayLength` could run into an error with `undefined` (`this.selected[0 -1] === undefined`). In this particular case it doesn't run into an error, but it's generally a safer approach)

Reply:
Will fix this and the other one ASAP.